services/monitoring/prometheus/prometheus-aws.yml

```diff
@@ -0,0 +1,38 @@
+scrape_configs:
+  - job_name: 'federate-computational-clusters-ec2'
+    honor_labels: true
+    metrics_path: '/federate'
+    params:
+      # we match all metrics for now
+      'match[]':
```

Comment:
I have no clue, are you sure this should be quoted?

Reply:
well it worked on my local machine. now I am not sure yet if it will work from an EC2 machine directly.